modules/api/src/server.cpp

```diff
@@ -1,5 +1,5 @@
 #include "server.h"
-#include "spdlog/spdlog.h"
+#include "../../external/spdlog/spdlog.h"
```

Comment:
Keep using `#include "spdlog/spdlog.h"`. Where the headers located should be configured in `cmake`'s `target_include_directories`.